sky/data/mounting_utils.py

```diff
@@ -49,9 +49,15 @@ def get_s3_mount_cmd(bucket_name: str,
 
 def get_gcs_mount_install_cmd() -> str:
     """Returns a command to install GCS mount utility gcsfuse."""
-    install_cmd = ('wget -nc https://github.com/GoogleCloudPlatform/gcsfuse'
+    install_cmd = ('ARCH=$(uname -m) && '
```

Comment:
Should we leave a quick TODO for goofys install_cmd too? Doesn't seem like goofys supports ARM, so we might need a different mounting utility in the future

Reply:
Good point! Added a specific exit code for it. We should probably switch to rclone for S3 as well : )